src/frontend/classic/theme/sidenav-hack.css

```diff
@@ -0,0 +1,58 @@
+.bx--side-nav {
```

Comment:
Why is this file called hack?
Seems to me like overwriting pre-defined classes from a theme, so i suggest a more appropriate name.

Reply:
It's a valid temporary fix because carbon (the base ibm project) doesn't allow theme customisation on the sidenav. It will live untill they code it.

Reply:
Like @Sheepux said, it's a temporary fix so for me the name is appropriate